test-data/unit/cmdline.test

```diff
@@ -130,30 +130,6 @@ two/mod/__init__.py: note: See https://mypy.readthedocs.io/en/stable/running_myp
 two/mod/__init__.py: note: Common resolutions include: a) using `--exclude` to avoid checking one of them, b) adding `__init__.py` somewhere, c) using `--explicit-package-bases` or adjusting MYPYPATH
 == Return code: 2
 
-[case testFlagsFile]
-# cmd: mypy @flagsfile
-[file flagsfile]
--2
-main.py
-[file main.py]
-def f():
-    try:
-        1/0
-    except ZeroDivisionError, err:
-        print err
-
-[case testConfigFile]
```

Comment:
Similar to above, this doesn't look Python 2 specific as such, but tests using `mypy.ini`. Please update the test case to use some other config file option.